windse/ProblemManager.py

```diff
@@ -971,6 +994,13 @@ def ComputeFunctional(self,inflow_angle=None):
         self.tf = self.ComputeTurbineForce(self.u_k,inflow_angle)
         self.u_k.assign(self.bd.bc_velocity)
 
+        # Only the actuator lines point "upstream" against the flow
+        # all other actuator forces need to be reversed to follow
+        # the convention used in the unsteady problem
+        # FIXME: We should be consistent about which way the turbine
+        # forces are oriented.
+        if self.farm.turbine_method != "alm":
+            self.tf *= -1.0
```

Comment:
@jefalon what do you think about standardizing the direction the turbine force points? I'd like to swap everything so that turbine forces point opposite the incoming flow, but recognize that ALM is the odd one out and it would be easier to bring it into line with the others.

Reply:
I think that's a good idea and having it opposite to flow is logical since it's suppose to be forcing it to slow down. Thinks to check to make sure this sign flip doesn't break anything is:
 - the sign of the tf term in the system functional for both the stabilized and Taylor hood steady solver
 - the sign on the power and 2d_power objective functions
 - probably some other place I can't think of

Reply:
We need to fix the opt output for plotting